package nestedExceptionHandling;

public class A {

	public void fun1() {
		System.out.println("inside fun1 of A");
	}
}
